src/other/Table.css

```diff
@@ -6,13 +6,18 @@
 }
 
 .other_Table th {
-    padding:5px 10px;
+	padding:5px 10px;
 	background-color:#1f77b4;
 	border-width: 1px;
 	border-style: solid;
 	border-color: #a9c6c9;
```

Comment:
@mlzummo thanks :) I think it fixes the issue! Don't really know what causes it.